modules/openapi-generator/src/main/resources/go/api_test.mustache

```diff
@@ -36,7 +36,12 @@ func Test_{{packageName}}_{{classname}}Service(t *testing.T) {
 		t.Skip("skip test")  // remove to run test
 
 		{{/-first}}
+        {{#isPrimitiveType}}
```

Comment:
looks like you're using a mix of tabs and spaces for indentation

what about replacing spaces above with tabs instead to make it consistent with the rest of the template?